packages/cli-poa/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@percy/cli-poa",
+  "version": "1.0.0",
```

```suggestion
  "version": "1.0.0",
```

this would be hooked with CLI release, so starting point would be current version.